docs/usage/dependency-pinning.md

```diff
@@ -26,11 +27,11 @@ If instead you "pin" your dependencies rather than use ranges, it means you use
 ## Why use ranges?
 
 For projects of any type, the main reason to use ranges is so that you can "automatically" get updated releases - which may even include security fixes.
-By "automatically", we mean that any time you run `npm install` you will get the very latest version matching your semver - assuming you're not using a lock file, that is.
+By "automatically", we mean that any time you run `npm install` you will get the very latest version matching your SemVer - assuming you're not using a lock file, that is.
 
 #### Tilde vs Caret
```

Comment:
By the way I noticed we're going from `h2` to `h4` in multiple places.
I think that's not what we want?